modelscope/pipelines/builder.py

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) Alibaba, Inc. and its affiliates.
```

Comment:
Please fix the unittest function case: tests.pipelines.test_human3d_render.Human3DRenderTest.test_run_modelhub

Delete: 'case_id': '3f2a7538253e42a8',
Add:  'case_id': '000039',

to pass the unittest run action.